packages/dev/docs/src/Layout.js

```diff
@@ -115,37 +129,105 @@ export function Layout({scripts, styles, pages, currentPage, publicUrl, children
-              <li className={classNames(sideNavStyles['spectrum-SideNav-item'], {[sideNavStyles['is-selected']]: p.name === currentPage})}>
-                <a className={sideNavStyles['spectrum-SideNav-itemLink']} href={p.url}>{path.basename(p.name, path.extname(p.name))}</a>
-              </li>
-            ))}
-          </ul>
-        </nav>
-        <main>
-          <article className={typographyStyles['spectrum-Typography']}>
-            <MDXProvider components={mdxComponents}>
-              <LinkProvider>
-                {children}
-              </LinkProvider>
-            </MDXProvider>
-          </article>
-          <ToC toc={toc} />
-        </main>
+        {children}
       </body>
     </html>
   );
 }
+
+function dirToTitle(dir) {
+  return dir
+    .split('/')[0]
+    .split('-')
+    .map(w => w.charAt(0).toUpperCase() + w.slice(1))
+    .join(' ');
+}
+
+function Nav({currentPageName, pages, publicUrl}) {
+  let isIndex = /index\.html$/;
+  let currentParts = currentPageName.split('/');
+  let currentDir = currentParts[0];
+    
+  pages = pages.filter(p => {
+    let pageParts = p.name.split('/');
+    let pageDir = pageParts[0];
+
+    // Pages within same directory (react-spectrum/Alert.html)
+    if (currentParts.length > 1) {
+      return currentDir === pageDir && !isIndex.test(p.name);
+    }
+
+    // Top-level index pages (react-spectrum/index.html)
+    if (currentParts.length === 1 && pageParts.length > 1 && isIndex.test(p.name)) {
+      return true;
+    }
+
+    // Other top-level pages
+    return !isIndex.test(p.name) && pageParts.length === 1;
+  });
+
+  let title = currentParts.length > 1 ? dirToTitle(currentPageName) : 'React Spectrum';
+
+  return (
+    <nav className={docStyles.nav}>
+      <header>
+        {currentParts.length > 1 &&
+          <a href="../index.html" className={docStyles.backBtn}>
+            <ChevronLeft />
+          </a>
+        }
+        <a href={publicUrl} className={docStyles.homeBtn}>
+          <svg viewBox="0 0 30 26" fill="#E1251B">
+            <polygon points="19,0 30,0 30,26" />
+            <polygon points="11.1,0 0,0 0,26" />
+            <polygon points="15,9.6 22.1,26 17.5,26 15.4,20.8 10.2,20.8" />
+          </svg>
+          <h2 className={typographyStyles['spectrum-Heading4']}>
+            {title}
+          </h2>
+        </a>
+      </header>
+      <ul className={sideNavStyles['spectrum-SideNav']}>
+        {pages.map(p => (
+          <li className={classNames(sideNavStyles['spectrum-SideNav-item'], {[sideNavStyles['is-selected']]: p.name === currentPageName})}>
+            <a className={sideNavStyles['spectrum-SideNav-itemLink']} href={p.url}>{p.title}</a>
+          </li>
+        ))}
+      </ul>
+    </nav>
+  );
+}
+
+function Footer() {
+  const year = new Date().getFullYear();
+  return (
+    <footer>
```

Comment:
this footer is (mostly) stolen from the spectrum sites, except i removed the cookie opt-out stuff since we dont have any tracking scripts installed.

Reply:
also, they seem to use a large script that's used by adobe.com -- i didnt want to include all that code just for a footer.